jekyll/_cci2/env-vars.md

```diff
@@ -43,7 +43,7 @@ Private environment variables enable you to store secrets safely, even when your
 ## Environment variable usage options
 {: #environment-variable-usage-options }
 
-CircleCI uses Bash, which follows the POSIX naming convention for environment variables. Valid characters include letters (uppercase and lowercase), digits, and the underscore. The first character of each environment variable must be a letter.
+CircleCI uses Bash, which follows the POSIX naming convention for environment variables. Valid characters include letters (uppercase and lowercase), digits, and the underscore. 
```

Comment:
This is correct, but incomplete. Quoting from https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html:

```
Environment variable names used by the utilities in the Shell and Utilities volume of POSIX.1-2017 consist solely of uppercase letters, digits, and the <underscore> ( '_' ) from the characters defined in [Portable Character Set](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap06.html#tagtcjh_3) and do not begin with a digit.
```

So we should say something along the lines of
```suggestion
CircleCI uses Bash, which follows the POSIX naming convention for environment variables. Valid characters include letters (uppercase and lowercase), digits, and the underscore. The first character of each environment variable must not be a digit.
```

This rule is in fact enforced in the CircleCI UI, I was not able to provision an environment variable name starting with a digit.

Reply:
Thank you @jtatarik! I just made a minor change to your suggestion